app/src/main/res/values/styles.xml

```diff
@@ -2,10 +2,8 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.NoActionBar">
```

Comment:
The dark theme looks good but curious why the change from the current theme?

Reply:
I like dark themes. That was literally the extent of my thinking :P
We can put the purple theme back in before accepting the PR if you want
or if you're ok with the dark theme, that's cool too. Whatever

Reply:
If we wanna use the dark theme: no changes to the PR required
If we wanna use the purple theme: I gotta undo some theme changes
If we wanna use both: That's cool too